## Chapter 2: Measuring Uncertainty

<b> Q2. What is the probability of rolling three six-sided dice and getting a value greater than 7?

<b>For Loop Example

In [1]:
count <- 0
for(roll1 in c(1:6)){
    for(roll2 in c(1:6)){
        for(roll3 in c(1:6)){
            count <- count + ifelse(roll1+roll2+roll3 > 7,1,0)
        }
    }
}

In [3]:
print(count)

[1] 181


In [4]:
count/216

[1] 0.837963

The probability of the rolls totaling more than 84.8%

<b> Single Line Example

In [9]:
sum(apply(expand.grid(c(1:6),c(1:6),c(1:6)),1,sum) > 7)

[1] 181

In [5]:
181/216

[1] 0.837963

<b> Q3. The Yankees are playing the Red Sox. You’re a diehard Sox fan and bet your friend they’ll win the game. You’ll pay your friend 30 dollars if the Sox lose and your friend will have to pay you only 5 dollars if the Sox win.

In [16]:
yanks_win = 30
sox_win = 5
odds = yanks_win/sox_win
print(odds)

[1] 6


<b> What is the probability you have intuitively assigned to the belief that the Red Sox will win?

In [18]:
p = odds/(1+odds)
p

[1] 0.8571429

<b>Probability you've placed on Redsox beating Yankees is 6/7 or about 85.7%